src/.defaultsForValidator.js

```diff
@@ -35,7 +35,8 @@ const defaults = {
       'content_type_parameter': 'error',
       'accept_type_parameter': 'error',
       'authorization_parameter': 'warning',
-      'required_param_has_default': 'warning'
+      'required_param_has_default': 'warning',
+      'ref_and_inline_parameter': 'error'
```

Comment:
Why is this getting changed? We're not adding any new rules here

Reply:
This was an accident 😅